src/test/java/io/confluent/idesidecar/restapi/application/SidecarInfoTest.java

```diff
@@ -54,134 +59,212 @@ public static SidecarInfo createSidecarInfo(TestInputs input) {
             "Windows 10 info with VS Code and version prefix",
-            "Windows 10",
-            "10.0.1904562",
+            "windows",
+            "10.1.X.Y",
+            "20.1.2",
             OperatingSystemType.Windows,
             new SidecarInfo.VsCode("20.1.2", "1.2.3"),
-            "v"
+            "v",
+            "Confluent-for-VSCode/v1.2.3 (https://confluent.io; support@confluent.io) sidecar/v%s (windows/%s)"
+                .formatted(
+                    sidecarInfo.version(),
+                    sidecarInfo.osArch())
         ),
         new TestInputs(
             "Windows 10 info without VS Code",
-            "Windows 10",
-            "10.0.1904562",
+            "windows",
+            "10.1.X.Y",
+            "20.1.2",
             OperatingSystemType.Windows,
             null,
-            null
+            null,
+            "Confluent-for-VSCode/vunknown (https://confluent.io; support@confluent.io) sidecar/v%s (windows/%s)"
+                .formatted(
+                    sidecarInfo.version(),
+                    sidecarInfo.osArch())
         ),
         // Windows 11
         new TestInputs(
             "Windows 11 info with VS Code",
-            "Windows 11",
-            "10.0.1904562",
+            "windows",
+            "10.1.X.Y",
+            "20.1.2",
             OperatingSystemType.Windows,
             new SidecarInfo.VsCode("20.1.2", "1.2.3"),
-            ""
+            "",
+            "Confluent-for-VSCode/v1.2.3 (https://confluent.io; support@confluent.io) sidecar/v%s (windows/%s)"
+                .formatted(
+                    sidecarInfo.version(),
+                    sidecarInfo.osArch())
         ),
         new TestInputs(
             "Windows 11 info with VS Code and version prefix",
-            "Windows 11",
-            "10.0.1904562",
+            "windows",
+            "10.1.X.Y",
+            "20.1.2",
             OperatingSystemType.Windows,
             new SidecarInfo.VsCode("20.1.2", "1.2.3"),
-            "v"
+            "v",
+            "Confluent-for-VSCode/v1.2.3 (https://confluent.io; support@confluent.io) sidecar/v%s (windows/%s)"
+                .formatted(
+                    sidecarInfo.version(),
+                    sidecarInfo.osArch())
         ),
         new TestInputs(
             "Windows 11 info without VS Code",
-            "Windows 11",
+            "windows",
             "10.1.X.Y",
+            "20.1.2",
             OperatingSystemType.Windows,
             null,
-            null
+            null,
+            "Confluent-for-VSCode/vunknown (https://confluent.io; support@confluent.io) sidecar/v%s (windows/%s)"
+                .formatted(
+                    sidecarInfo.version(),
+                    sidecarInfo.osArch())
         ),
         // Other
         new TestInputs(
-            "Solaris info with VS Code",
-            "Solaris 4",
+            "Unix info with VS Code",
+            "unix",
             "4.1.X.Y",
```

Comment:
Based on the `TestInput` record members, the third argument corresponds to the `osArch` and not `osVersion`. This needs to be updated to reflect a valid osArch.